AUDITING.md

```diff
@@ -0,0 +1,75 @@
+# Auditing
+
+This document defines how Platform Chaos audits chaos events. It lists what is tracked and how it is formatted.
+
+## Tracked Data
+
+When a chaos event is terminated through an *API*, *CLI*, or *Extension*, the following properties are recorded and exported through a JSON object.
```

Comment:
this calls out terminated only - should this reference other conditions as well?

Reply:
yes